CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 
 ### Fixed
 
+- Fixed a change in behavior of normalize host brand string that breaks
+  Firecracker on external instances.
```

Comment:
Not a big fan of this string, suggested something like "Fixed a bug which prevented to start Firecracker microVMs if the CPU model string did not match our assumptions" but this is a simple backport so i do not know if it is even worst having not matching changelog between main and the release branch